Viktoriia/TicTacToe/main.cpp

```diff
@@ -0,0 +1,142 @@
+#include<iostream>
+using namespace std;
+
+void ShowImage(char mas[][3], int size);
+void Cross(char mas[][3], int size);
+void Zero(char mas[][3], int size);
+bool FieldFilling(char mas[][3], int size, char possCross, char possZero);
+bool CheckWin(char mas[][3], int size);
+bool EndGame(char mas[][3], int size);
+
+int main()
+{
+    const int HEIGHT = 3;
+    const int WIDTH = 3;
+    char arr[HEIGHT][WIDTH] = { { '1','2','3' }, { '4', '5','6' }, { '7','8','9' } };
+    ShowImage(arr, 3);
+    Cross(arr,3);
+	return 0;
+}
+
+void ShowImage(char mas[][3], int size) // ���� ��������� ����
+{
+        for (int i = 0; i < size; i++)
+        {
+            for (int j = 0; j < size; j++)
+            {
+                cout << "    " << mas[i][j] << "     |    " << mas[i][j + 1] << "     |    " << mas[i][j + 2] << "    \n" << "----------|----------|---------\n";
+                break;
+            }
+        }
+}
+
+void Cross(char mas[][3], int size) 
```

Comment:
Actually recursion is not the best solution here because it takes um more memory and what is more important it leads to duplication of big code portions (`Cross` and `Zero` functions are 90% identical).